background_scripts/main.js

```diff
@@ -295,6 +356,28 @@ const BackgroundCommands = {
   moveTabLeft: moveTab,
   moveTabRight: moveTab,
 
+  async setZoom({ tabId, registryEntry }) {
+    const zoomLevel = registryEntry.optionList[0] ?? 1;
+    const newZoom = parseFloat(zoomLevel);
+    // Chrome breaks on values below 0.01 and 10 or above.
```

Comment:
Note that Firefox just prints an error to the console when we try to zoom to a value outside of its range (0.3 to 5.0). Everything keeps working fine, and Firefox simply ignores the zoom. We can add a special case to change our handling of this if you like.